### ISSUE TITLE
Questions about label transfer a continuous variable with TransferData. 

### ISSUE THREAD

**Issue by Hsu-Che-Wei** [satijalab/seurat#8651]
Hi,

In the Seurat documentation, it claims that the function TransferData can be used to transfer continuous variable. I would like to transfer pseudotime values from a reference single cell data to a new sample, and would like to do it with TransferData. However, in the documentation, one should provide a "matrix" from one of the "assays" to the refdata instead of a vector of metadata column where pseudotime values are stored. My question is, where and how can I specify the pseudotime value to be transferred? 

In summary, could ypu please show me how to apply TransferData if I want to transfer a continuous variable such as pseudotime? 

Thank you very much!

**Comment by igrabski** [comment]
Hi, can you show the code you tried and what the error was?
